ksp_plugin_adapter/map_node_pool.cs

```diff
@@ -45,11 +45,12 @@ public class SingleProvenancePool {
         new List<KSP.UI.Screens.Mapview.MapNode>(MaxNodesPerProvenance);
   }
 
-  public MapNodePool(Func<bool> show_unpinned) {
+  public MapNodePool(Func<bool> show_unpinned, Func<bool> can_hover) {
```

Comment:
The addition of a 2nd callback here feels a bit clunky but I am not sure what to suggest instead.  Let's see what egg has to say.

Reply:
A single callback returning a 2-tuple/struct?